setup.py

```diff
@@ -30,6 +30,7 @@
     },
 
     install_requires=[
-        'pylint>=1.4.0',
+        'pylint==1.4.4',
+        'pylint-django==0.6.1',
```

Comment:
I've changed requirements.txt to not mention pylint, instead to install setup.py.